src/Lykke.Cqrs/Middleware/CommandLoggingInterceptor.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Common;
+using Common.Log;
+using Lykke.Common.Log;
+using Lykke.Cqrs.Abstractions.Middleware;
+
+namespace Lykke.Cqrs.Middleware
+{
+    public class CommandLoggingInterceptor : ICommandInterceptor
+    {
+        private readonly ILog _log;
+        private readonly Dictionary<Type, Action<ILog, object, object>> _customLoggingActionsMap;
+
+        [Obsolete]
+        public CommandLoggingInterceptor(ILog log, Dictionary<Type, Action<ILog, object, object>> customLoggingActionsMap = null)
+        {
+            _log = log;
+            _customLoggingActionsMap = customLoggingActionsMap;
+        }
+
+        public CommandLoggingInterceptor(ILogFactory logFactory, Dictionary<Type, Action<ILog, object, object>> customLoggingActionsMap = null)
```

Comment:
Will be nice to add custom delegate with meaningful argument names instead of Action<>.

Reply:
And I don't think that this ```customLoggingActionsMap``` needed at all in this generic interceptor. If someone will need to log particular messages in some special way, then it can just implement his own implementation of interceptor and aggregate given implementation to use it as default. In this case generic implementation will be clear and in specific implementation we can make whatever we need without any constraints built in the ```CommandLoggingInterceptor```.